readme.md

```diff
@@ -5,7 +5,7 @@
 ### System packages
 
 ```
-sudo apt install nginx redis
+sudo apt install nginx redis git tmux
```

Comment:
Yep, gcc is also required otherwise installing grpcio from requirements.txt fails.

Reply:
thanks, are you testing on Debian 10 as well?

Reply:
Nop, Centos 8, but the package name should be the same.

Reply:
ok, added gcc